packages/tests/tests/sdk/application_version.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { createApplicationNode, WakuNode } from "@waku/sdk";
+import {
+  contentTopicToPubsubTopic,
+  ensureValidContentTopic
+} from "@waku/utils";
+import { expect } from "chai";
+
+describe("SDK: Creating by Application and Version", function () {
+  const ContentTopic = "/myapp/1/latest/proto";
+
+  it("given an application and version, creates a waku node with the correct pubsub topic", async function () {
+    const contentTopic = ensureValidContentTopic(ContentTopic);
+    const waku = await createApplicationNode(
+      contentTopic.application,
```

Comment:
At this point I am thinking if we need a class for it:
```ts
class ContentTopic {
  static public fromString(value: string): ContentTopic {
    return new ContentTopic({ value });
  }

  constructor({ value, application, version }) {}

  get application() {}
  get version() {}
  get encoding() {}
}
```

Reply:
@adklempner if you agree with approach we can follow up with this in a separate issue. Can be linked to https://github.com/waku-org/pm/issues/143